pkg/cluster/internal/kubeadm/config.go

```diff
@@ -349,9 +288,7 @@ func Config(data ConfigData) (config string, err error) {
 
 	// assume the latest API version, then fallback if the k8s version is too low
 	templateSource := ConfigTemplateBetaV2
-	if ver.LessThan(version.MustParseSemantic("v1.12.0")) {
-		templateSource = ConfigTemplateAlphaV2
```

Comment:
Yeah, they aren't running, also 1.15 is close to be ready... No strong feelings but we should move on sooner or later :)

Reply:
SGTM

Reply:
We only did because one of the tests is broken and officially Kubernetes CI won't support it. We should make best effort to support feasible versions.

Reply:
I dropped the 1.11 in a separate commit, let me remove that commit and figure out the code for the v1alpha2 config files